src/fancy-tools/brushEraserTool.js

```diff
@@ -0,0 +1,144 @@
+import EVENTS from './../events.js';
+import external from './../externalModules.js';
+import baseBrushTool from './../base/baseBrushTool.js';
+import toolColors from './../stateManagement/toolColors.js';
+// Utils
+import getCircle from './shared/brushUtils/getCircle.js';
+import { drawBrushPixels } from './shared/brushUtils/drawBrush.js';
+import KeyboardController from './shared/KeyboardController.js';
+// State
+import { getToolState, addToolState } from './../stateManagement/toolState.js';
+import { getters } from './../store/index.js';
+
+export default class extends baseBrushTool {
+  constructor (name = 'brushEraser') {
+    super({
+      name,
+      supportedInteractionTypes: ['mouse'],
+      configuration: defaultBrushConfiguration()
+    });
+
+    const keyBinds = this.configuration.keyBinds;
+
+    this._keyboardController = new KeyboardController(this, keyBinds);
```

Comment:
If we go this route, we can probably have the `super`/`base` take care of instantiating the KeyboardController if keyBinds are specified?

Reply:
That's a good idea. I tried to be less intrusive to core whilst these were the only tools using hotkeys. I guess I could implement this now.

Reply:
That's a good idea. I tried to be less intrusive to core whilst these were the only tools using hotkeys. I guess I could implement this now.

Reply:
I've put this in baseTool. Now all a tool actually has to do to hook up a keybinds is attach a keybind object to config ala:
```
keyBinds: {
  methodOne: '+',
  methodTwo: '-'
}
```

that will hookup methodOne() and methodTwo() to + and -, respectively.

Through the use of pretty new setters, setting new keybinds is just this:
```
const newKeyBinds: {
  methodOne: 'a',
  methodTwo: 'b'
}

tool.keyBinds = newKeyBinds;
```

ofc if no config.keyBinds object is found, no KeyboardController is made.

Reply:
I think that makes sense and is a good solution. If we want to account for collisions, we can just modify the `KeyboardController`, and if _any tool_ wants to leverage keybinding, they can do so with simple configuration.